### ISSUE TITLE
Fixing function call brackets indentation

### ISSUE THREAD

**Issue by ostrolucky** [squizlabs/PHP_CodeSniffer#2076]
Nothing seems to fix following case:
```php
return $this->render('Admin/League/form.html.twig', [
                'form' => $form->createView(),
                            ]);
```

Expected:
```php
return $this->render('SuperAdmin/QuestionPack/Question/add.html.twig', [
    'form' => $form->createView(),
]);
```

Maybe ScopeIndentSniff can do that? Or is it outside of its reponsibilities?

**Comment by gsherwood** [comment]
This is no ScopeIndent's job because it is hard to have a fixed standard for how control structures and function calls should be indented.

The closest I think you'll get with the built-in sniffs is with the PEAR standard (specifically PEAR.Functions.FunctionCallSignature) but it's going to be doing other things you don't want. So I don't think you can currently do this without a new sniff.
```
$ phpcbf temp.php --standard=PEAR --sniffs=PEAR.Functions.FunctionCallSignature
$ cat temp.php
<?php
return $this->render(
    'Admin/League/form.html.twig', [
                'form' => $form->createView(),
    ]
);
```
Note that the array indent is weird here because no array sniffs are running.

**Comment by ostrolucky** [comment]
Yeah I know that one, but I don't want to break all arguments to new line. Is such new sniff reasonable to ask for, then? Is this something core team wishes to have?

**Comment by gsherwood** [comment]
Yes, being able to just use the indent checks without the other ones would be helpful. I'll make this a feature request.

**Comment by ostrolucky** [comment]
https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3926
